charts/rancher-gke-operator/1.0.500/values.yaml

```diff
@@ -0,0 +1,11 @@
+global:
+  systemDefaultRegistry: ""
+
+eksOperator:
+  image:
+    repository: rancher/gke-operator
+    tag: v1.0.5
```

Comment:
Same, version should probably be `1.0.0`.

Reply:
done